subjects/devops/plus/README.md

```diff
@@ -0,0 +1,43 @@
+## plus
+
+### Instructions
+
+In this exercise, you will make a script that will take two arguments from the command line, add them and output the result.
+
+### Usage
+
+```console
```

Comment:
We are not using the cat -e flag like in the go and etc... so if you decide to use it here add a hint to the hint section explaining the flag as its the first time it will appear.

Reply:
That's right. I'll remove it since it's not needed. Thank you.